packages/vscode-extension/src/debug/teamsfxTaskHandler.ts

```diff
@@ -46,13 +52,44 @@ function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): void {
     }
 }
 
+function onDidStartDebugSessionHandler(event: vscode.DebugSession): void {
+    if (ext.workspaceUri && isWorkspaceSupported(ext.workspaceUri.fsPath)) {
+        const debugConfig = event.configuration;
+        if (debugConfig &&
+            debugConfig.name &&
+            (debugConfig.url || debugConfig.port) && // it's from launch.json
+            !debugConfig.postRestartTask) // and not a restart one
+        {
+            // send f5 event telemetry
+            try {
+                ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DebugStart, {
+                    [TelemetryProperty.DebugSessionId]: event.id,
+                    [TelemetryProperty.DebugType]: debugConfig.type,
+                    [TelemetryProperty.DebugRequest]: debugConfig.request,
+                    [TelemetryProperty.DebugPort]: debugConfig.port + "",
+                    [TelemetryProperty.DebugRemote]: debugConfig.name?.includes("Remote") || debugConfig.name?.includes("remote") ? "true" : "false"
```

Comment:
How abount adding a check according to `url` in case user modify the configuration name?

Reply:
Add app id check to determine if it's a remote debug session